qemu/tests/block_with_iommu.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from virttest import utils_misc
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test the vIOMMU platform.
+
+    Steps:
+        1. Add "intel_iommu=on" to kernel line of q35 guest.
+        2. Boot a guest with virtio-scsi with iommu_platform=on.
+        3. Verify IOMMU enabled in the guest.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def verify_iommu_enabled():
+        """ Verify whether the iommu is enabled. """
+        error_context.context(
+            'Verify whether IOMMU is enabled in the guest.', logging.info)
+        for key_words in params['check_key_words'].split(';'):
+            output = session.cmd_output("journalctl -k | grep -i \"%s\"" % key_words)
+            if not output:
+                test.fail("No found the info \"%s\" "
+                          "from the systemd journal log." % key_words)
+            logging.debug(output)
+
+    if utils_misc.get_cpu_vendor(verbose=False) != 'GenuineIntel':
```

Comment:
get_cpu_vendor move to virttest.cpu. please use latest avocado codes. e.g. cpu.get_cpu_vendor()

Reply:
Thanks for your comments, have updated it.